src/main/webapp/app/exam/participate/exam-navigation-bar/exam-navigation-bar.component.ts

```diff
@@ -5,6 +5,8 @@ import { CustomBreakpointNames } from 'app/shared/breakpoints/breakpoints.servic
 import * as moment from 'moment';
 import { Moment } from 'moment';
 import { IconProp } from '@fortawesome/fontawesome-svg-core';
+import { ExamParticipationService } from 'app/exam/participate/exam-participation.service';
+import { Submission } from 'app/entities/submission.model';
```

Comment:
`'Submission' is defined but never used`

Reply:
Done